Announcements/_posts/The Blue Pill Release/2021-08-03-Blue-Pill-Release.md

```diff
@@ -4,12 +4,12 @@ title:  "Blue Pill Release"
 categories: [ Announcements ]
 image: ./Blue-Pill-Release/thebluepill.jpg
 author: Yearn
-translator:
+translator: gateway-504
 publish: true
 ---
 
-Last week we announced The Blue Pill book and gave early Yearn supporters a first look via a limited series of NFTs.
+Minggu lalu kami mengumumkan buku The Blue Pill dan memberikan pandangan pertama kepada pendukung awal Yearn melalui seri terbatas NFT.
```

Comment:
Apakah lebih baik `seri NFT terbatas` bukan `seri terbatas NFT`?

```diff
- Minggu lalu kami mengumumkan buku The Blue Pill dan memberikan pandangan pertama kepada pendukung awal Yearn melalui seri terbatas NFT.
+ Minggu lalu kami mengumumkan buku The Blue Pill dan memberikan pandangan pertama kepada pendukung awal Yearn melalui seri NFT terbatas.
```

Reply:
Bukan,,yang betul "seri terbatas NFT"

Reply:
Thank you